yayson/yayson.d.ts

```diff
@@ -0,0 +1,82 @@
+// Type definitions for yayson v2.0.1
+// Project: https://github.com/confetti/yayson
+// Definitions by: David Wood <https://github.com/Codesleuth>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'yayson' {
+  class Store {
+    sync(obj: Object): void;
+    sync<T>(obj: Object): T;
+
+    find(type: string, id: string): any;
+    find<T>(type: string, id: string): T;
+
+    findAll(type: string): any[];
+    findAll<T>(type: string): T[];
+
+    remove(type: string, id: string): void;
+    remove(type: string): void;
+
+    reset(): void;
+
+    records: Array<any>;
+    relations: { [key: string]: any };
+  }
+
+  interface Adapter {
+    new (): any;
```

Comment:
This isn't needed, I'll remove it. It _is_ newable but it's never intended to be used that way.